flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/utils/AvroTestUtils.java

```diff
@@ -102,6 +104,13 @@ public final class AvroTestUtils {
                         .setTypeTimestampMillis(Instant.parse("2014-03-01T12:12:12.321Z"))
                         .setTypeTimestampMicros(
                                 Instant.ofEpochSecond(0).plus(123456L, ChronoUnit.MICROS))
+                        .setTypeLocalTimestampMillis(
+                                LocalDateTime.ofInstant(
```

Comment:
nit: Please use `LocalDateTime.parse("2014-03-01T12:12:12.32")` to construct a `LocalDateTime`.